zerver/openapi/zulip.yaml

```diff
@@ -39,6 +39,138 @@ security:
 # Endpoint definitions
 #######################
 paths:
+  /bot_storage:
+    get:
+      operationId: get-storage
+      summary: Get Bot storage
+      tags: ["server_and_organizations"]
+      description: |
+        Get the Bot storage
+        `GET {{ api_url }}/v1/bot_storage/get_storage`
+        **Changes**: New in Zulip 6.0 (feature level 125).
+      parameters:
+        - name: keys
+          in: query
+          description: List of storage keys.
+          required: False
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+              example: ["key1"]
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - $ref: "#/components/schemas/SuccessDescription"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      storage:
+                        type: object
+                        description: |
+                          A list of storage items
+                        additionalProperties:
+                          type: string
+                          description: |
+                            Returns storage Keys and its value
+                    example:
+                      {
+                        "msg": "",
+                        "storage": {"key1": "value1", "key2": "value2"},
+                        "result": "success",
+                      }
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonError"
+                  - example: {"msg": "Key does not exist.", "result": "error"}
+                    description: |
+                      An example JSON response for when a `get` operation is requested for list of
+                      storage were keys don't exist
+    put:
+      operationId: update-storage
+      summary: Update Bot Storage
+      tags: ["server_and_organizations"]
+      description: |
+        Update the Bot Storage
+        `PUT {{ api_url }}/v1/bot_storage/update_storage
```

Comment:
Same issue here as above with line breaks and below in the delete documentation.